docs/book/src/developer/e2e.md

```diff
@@ -198,6 +198,65 @@ long term success of the project.
 The [test E2E package] provides examples of how this can be achieved by implementing a set of reusable
 test specs for the most common Cluster API use cases.
 
+## Running the e2e tests locally
+
+Usually, the e2e tests are executed by Prow either pre-submit (on PRs) or periodically on certain branches 
+(e.g. the default branch). Those jobs are defined in the kubernetes/test-infra repository in [config/jobs/kubernetes-sigs/cluster-api](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/cluster-api). For development and debugging 
+those tests can also be executed locally. 
+
+### Via command line
+
+To run the tests locally via command line we can just take a look at the Job configuration and execute the same locally. 
+For example to run [pull-cluster-api-e2e-main](https://github.com/kubernetes/test-infra/blob/49ab08a6a2a17377d52a11212e6f1104c3e87bfc/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml#L113-L140) just execute:
+
+```bash
+GINKGO_FOCUS="\[PR-Blocking\]" ./scripts/ci-e2e.sh
+```
+
+### Via IDE
```

Comment:
This will be very useful - I could have used these instructions a while ago. :)

Reply:
Hope so :). 

I didn't go into too much details. Some of this config is different for different Jobs, e.g. the upgrade Jobs. But I think as everything can be inferred from the Prow Job config it's probably the right level. Otherwise it would get outdated very fast.

Reply:
Yeah, good call. We can always expand on this if we find useful information to add later. At least this gives a good starting place.